test-requirements.txt

```diff
@@ -13,3 +13,4 @@ asttokens
 responses
 pysocks
 ipdb
+mockupdb
```

Comment:
I added this in a different commit because I ran tests without `tox` (https://github.com/getsentry/sentry-python/blob/master/CONTRIBUTING.md#running-tests) and it failed because the requirement wasn't defined